src/components/templates/AnimateMap/components/FirebarrelWidget/FirebarrelWidget.tsx

```diff
@@ -126,13 +122,10 @@ export const FirebarrelWidget: React.FC<FirebarrelWidgetProps> = ({
 
         return (
           <div
-            key={participant.participant.identity}
+            key={participant.participant.sparkleId}
             className="firebarrel-room__participant"
           >
-            <VideoParticipant
-              participant={participant.participant}
-              participantUser={participant.user}
-            />
+            <VideoCommsParticipant participant={participant.participant} />
```

Comment:
Wait, is this widget a copy of the VideoParticipant component? Same spam spam spam spam....